src/Umbraco.Web.BackOffice/Controllers/MemberTypeController.cs

```diff
@@ -150,17 +152,16 @@ public IActionResult DeleteById(int id)
     ///     be looked up via the db, they need to be passed in.
     /// </param>
     /// <returns></returns>
-    public IActionResult GetAvailableCompositeMemberTypes(
-        int contentTypeId,
-        [FromQuery] string[] filterContentTypes,
-        [FromQuery] string[] filterPropertyTypes)
+    [HttpPost]
```

Comment:
Changing the http method for a public method is a breaking change.

Reply:
Also: you've defined the http method as `Post` but the method name is prefixed with `Get` - this is inconsistent. Looking at the use case here, you are indeed just getting data back so a `Get` is appropriate here, you're not posting back processed data that needs persisting.